src/Components/ArtworkFilter/ArtworkFilterCreateAlert.tsx

```diff
@@ -23,55 +21,45 @@ export const ArtworkFilterCreateAlert: FC<ArtworkFilterCreateAlertProps> = ({
   renderButton,
   children,
 }) => {
-  const { entity } = useSavedSearchAlertContext()
-  const { aggregations, filters: rawFilters } = useArtworkFilterContext()
-  const filters = usePrepareFiltersForPills()
+  const { aggregations } = useArtworkFilterContext()
   const newAlertModalEnabled = useFeatureFlag("onyx_artwork_alert_modal_v2")
+  const filters = usePrepareFiltersForPills()
+  const { entity } = useSavedSearchAlertContext()
 
   const criteria = getSearchCriteriaFromFilters(entity, filters)
   const metric = filters?.metric ?? DEFAULT_METRIC
 
-  const initialCriteria: SearchCriteriaAttributes = {}
-  if (entity?.owner?.type === OwnerType.artist) {
-    initialCriteria.artistIDs = [entity.owner.id]
-  }
-
-  const { alertComponent, showAlert } = useAlert({
-    initialCriteria: {
-      ...rawFilters,
-      ...initialCriteria,
-    },
-  })
-
-  const { clickedCreateAlert } = useAlertTracking()
+  // If there is no entity then we don't want to create an alert
+  if (isEmpty(entity)) return null
```

Comment:
Still relying on this entity concept to conditionally render on the right pages, but I'm hoping we can refactor this out soon.